readme_de.md

```diff
@@ -0,0 +1,260 @@
+| show_owner       | boolean   | zeigt den Namen des Besitzers in der Repo-Karte      | N/A                   | false               | N/A                     |
+| show_icons       | boolean   | zeige Icons an                                       | false                 | N/A                 | N/A                     |
+| theme            | string    | setze eingebaute themes                              | 'default'             | 'default_repocard'  | 'default'               |
+| cache_seconds    | number    | manuelles setzen der Cachezeiten                     | 1800                  | 1800                | '1800'                  |
+| hide_langs_below | number    | verberge Sprachen unter einem bestimmten Schwellwert | N/A                   | N/A                 | undefined               |
+
+> Hinweis bzgl. des Caches: Wenn die Anzahl der Forks und Stars geringer als 1Tsd ist, haben die Repo-Cards eine default Cachezeit von 30 Minuten (1800 Sekunden), ansonsten beträgt diese 2 Stunden (7200 Sekunden). Außerdem ist der Cache auf eine Minimum von 30 Minuten und ein Maximum von 24 Stunden begrenzt.
+
+# GitHub Extra Pins
+
+GitHub extra pins ermöglicht es, mit Hilfe eines GitHub-Readme-Profiles, mehr als 6 Repositories in deinen Profil anzuzeigen.
+
+Und Bääm! Du bist nicht mehr auf 6 pinned Repositories limitiert.
+
+### Benutzung
+
+Füge diesen Code in deine Readme-Datei ein und passe die Links an.
+
+Endpunkt: `api/pin?username=anuraghazra&repo=github-readme-stats`
+
+```md
+[![ReadMe Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats)](https://github.com/anuraghazra/github-readme-stats)
+```
+
+### Beispiele
+
+[![ReadMe Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats)](https://github.com/anuraghazra/github-readme-stats)
+
+Benutze die [show_owner](#anpassungenpersonalisierung) Variable, um den Usernamen des Repo Eigentümers anzuzeigen.
+
+[![ReadMe Card](https://github-readme-stats.vercel.app/api/pin/?username=anuraghazra&repo=github-readme-stats&show_owner=true)](https://github.com/anuraghazra/github-readme-stats)
+
+# Top Programmiersprachen Card
+
+Die Top Programmiersprachen Card visualisiert die am meisten benutzten Programmiersprachen eines GitHub-Nutzers.
+
+_HINWEIS: Die Top Programmiersprachen treffen keine Aussage über persönliche Fähigkeiten oder der gleichen, es ist lediglich eine auf den GitHub-Statistiken des Nutzers basierende Kennzahl welche Programmiersprache wie häufig verwendet wurde._
+
+### Benutzung
+
+Füge diesen Code in deine Readme-Datei ein und passe die Links an.
+
+Endpunkt: `api/top-langs?username=anuraghazra`
+
+```md
+[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
+```
+
+### Verberge Programmiersprachen unter einem bestimmten Schwellwert
```

Comment:
btw i really don't know german but is the word "Programmiersprachen" means "Languages" or "Programming languages" ???

If its "Programming languages" then we need to change to "Languages"

Reply:
Yes, it is "Programming languages"

Reply:
It should be just "Top Languages" @christian-draeger